packages/sdk/src/types/Distributions.ts

```diff
@@ -0,0 +1,168 @@
+interface DistributionsPublicationsVoucherDiscountUnit {
+	type?: 'UNIT'
+	unit_off?: number
+	effect?: string
+}
+interface DistributionsPublicationsVoucherDiscountAmount {
+	type?: 'AMOUNT'
+	amount_off?: number
+}
+interface DistributionsPublicationsVoucherDiscountPercent {
+	type?: 'PERCENT'
+	percent_off?: number
+}
+
+interface DistributionsPublicationsVoucher {
+	code?: string
+	object?: 'voucher'
+	campaign?: string
+	gift?: {
+		amount?: number
+		balance?: number
+	}
+	loyalty_card?: {
+		amount?: number
+		balance?: number
+	}
+	is_referral_code?: boolean
+	discount?:
+		| DistributionsPublicationsVoucherDiscountUnit
+		| DistributionsPublicationsVoucherDiscountAmount
+		| DistributionsPublicationsVoucherDiscountPercent
+}
+
+interface DistributionsPublicationsCreateResponseVoucher {
+	id?: string
+	code?: string
+	campaign?: string
+	campaign_id?: string
+	category?: string
+	type?: string
+	discount?:
+		| DistributionsPublicationsVoucherDiscountUnit
+		| DistributionsPublicationsVoucherDiscountAmount
+		| DistributionsPublicationsVoucherDiscountPercent
+	gift?: {
+		amount?: number
+		balance?: number
+	}
+	loyalty_card?: {
+		amount?: number
+		balance?: number
+	}
+	start_date?: string
+	expiration_date?: string
+	validity_timeframe?: string
+	validity_day_of_week?: string
+	active?: boolean
+	additional_info?: string
+	metadata?: Record<string, any>
+	assets?: {
+		qr?: {
+			id?: string
+			url?: string
+		}
+		barcode?: {
+			id?: string
+			url?: string
+		}
+	}
+	is_referral_code?: boolean
+	created_at?: string
+	updated_at?: string
+	holder_id?: string
+	object?: string
+	publish?: {
+		object?: string
+		count?: number
+		url?: string
+	}
+	redemption?: {
+		object?: string
+		quantity?: number
+		redeemed_quantity?: number
+		url?: string
+	}
+}
+// Eddy's comment (regarding how to set up filters type) is not quite working here - as we can have multiple `conditions` inside the `filter_condition`. I did not explained it correctly
+
+export interface DistributionsPublicationsListParams {
+	limit?: number
+	page?: number
+	order?: 'id' | 'voucher_code' | 'tracking_id' | 'customer_id' | 'created_at' | 'channel'
+	campaign?: string
+	customer?: string
+	voucher?: string
+	result?: string
+	voucher_type?: string
+	is_referral_code?: boolean
+	filters?: {
+		junction?: string
+		[filter_condition: string]: any
```

Comment:
Use `[filter_condition: string]: $FixMe` type instead, so we can come back to it later to fix it. 

The `$FixMe` type is an alias for `any` ( `type $FixMe = any` ), the difference is that `$FixMe` implies `any` is used but needs to be fixed later, but explicitly specifying `any` implies the change was intentional and doesn't need to be fixed.

Let's use that as a convention (I should have added a `Conventions` section to `Contributing.md` 😅 )

Reply:
Will do, thanks!